draft-packages/likert-scale-legacy/docs/LikertScaleLegacy.stories.tsx

```diff
@@ -1,12 +1,15 @@
 import React, { useState } from "react"
+import { Story } from "@storybook/react"
 import {
   LikertScaleLegacy,
   Scale,
   ScaleItem,
 } from "@kaizen/draft-likert-scale-legacy"
-import { Heading } from "@kaizen/typography"
+import { Box } from "@kaizen/component-library"
+import { Heading } from "../../../packages/typography"
```

Comment:
```suggestion
import { Heading } from "@kaizen/typography"
```

We probably want to keep this?